### PR TITLE
format and fix audio nodes and launch files

### PR DESCRIPTION
**Pull request by IceSentry**

### CODE REVIEW

File: audio_capture/src/audio_capture.cpp [hunk 1]
```diff
@@ -1,228 +1,230 @@
-#include <stdio.h>
```
Comment (lvanasse) [inline audio_capture/src/audio_capture.cpp]: Alright lol, my guess is that you apply the clang format on those files ?

**Review by lvanasse — APPROVED** [review]
Maybe revert the formatting, but the launch files looks fine to me

File: audio_play/src/audio_play.cpp [hunk 2]
```diff
@@ -129,7 +129,7 @@ namespace audio_transport
 
 int main (int argc, char **argv)
 {
-  ros::init(argc, argv, "audio_play");
+  ros::init(argc, argv, "audio_play", ros::init_options::AnonymousName);
```
Comment (lvanasse) [inline audio_play/src/audio_play.cpp:132]: Do you mind, giving me the rationale behind this modification ?

Comment (IceSentry) [reply audio_play/src/audio_play.cpp:132]: http://wiki.ros.org/roscpp/Overview/Initialization%20and%20Shutdown#Initializing_the_roscpp_Node
> node_name 
> This is the name that will be assigned to your node unless it's overridden by one of the remapping arguments. Node names must be unique across the ROS system. If a second node is started with the same name as the first, the first will be shutdown automatically. **In cases where you want multiple of the same node running without worrying about naming them uniquely, you may use the init_options::AnonymousName option described below.**

http://wiki.ros.org/roscpp/Overview/Initialization%20and%20Shutdown#Initialization_Options
> ros::init_options::AnonymousName
> Anonymize the node name. Adds a random number to the end of your node's name, to make it unique.

Comment (IceSentry) [reply audio_play/src/audio_play.cpp:132]: In case someone else forgets to rename the node

Comment (lvanasse) [reply audio_play/src/audio_play.cpp:132]: But does that impact the calling of the node when trying to communicate with it ? If it adds a random number, would it be impossible to know what will be the next number when you'll try to communicate with it ?

Comment (IceSentry) [reply audio_play/src/audio_play.cpp:132]: We communicate with topics, it doesn't matter what the node name is. At least not here.

Comment (lvanasse) [reply audio_play/src/audio_play.cpp:132]: Alright thank you

**Pull request merged**